<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServoCalibrationDialog</class>
 <widget class="QDialog" name="ServoCalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калибровка серв</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnOpen">
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <row>
      <property name="text">
       <string>S0</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>S1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>S2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>S3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>S4</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Откр.Угол</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Закр.Угол</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>Прочитать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnProg">
       <property name="text">
        <string>Записать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
